meta/recipes-connectivity/openssh/openssh_8.3p1.bb

```diff
@@ -5,11 +5,10 @@ Ssh (Secure Shell) is a program for logging into a remote machine \
 and for executing commands on a remote machine."
 HOMEPAGE = "http://www.openssh.com/"
 SECTION = "console/network"
-LICENSE = "BSD"
-LIC_FILES_CHKSUM = "file://LICENCE;md5=429658c6612f3a9b1293782366ab29d8"
+LICENSE = "BSD & ISC & MIT"
+LIC_FILES_CHKSUM = "file://LICENCE;md5=18d9e5a8b3dd1790d73502f50426d4d3"
 
-# openssl 1.1 patches are proposed at https://github.com/openssh/openssh-portable/pull/48
-DEPENDS = "zlib openssl10"
+DEPENDS = "zlib openssl"
```

Comment:
Can you add why you dropped the dependency on virtual/crypt in 9d81605f7dc3df4b83f3a4ae2d07d0a622518920 and 1609b06f1586b2e392ceb17b209fe281888e4b0c?

Reply:
These are patches that add a dependency on a virtual component that doesn't exist in sumo, so there is nothing there to depend on. An artifact of upgrading some components but not all.